highlighters/main/main-highlighter.zsh

```diff
@@ -153,19 +153,17 @@ _zsh_highlight_main_calculate_fallback() {
 #
 # If $2 is 0, do not consider aliases.
 #
+# If $3 is 0, do not consider reserved words.
+#
```

Comment:
A call to this function was added to master after this branch was crested. See whether it should be updated.